state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -1616,6 +1706,13 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 		refund_address: TargetChainAccount<T, I>,
 		deposit_fetch_id: Option<<T::TargetChain as Chain>::DepositFetchId>,
 	) {
+		if let Some(limit) = T::FetchesTransfersLimitProvider::maybe_transfers_limit() {
+			// In case we don't have enough nonces we put the tx back to be retried next block
+			if limit.is_zero() {
+				ScheduledTransactionsForRejection::<T, I>::append(tx);
+				return;
+			}
+		}
```

Comment:
This function has a comment saying that it doesn't do `maybe_transfers_limit()`. But now it does, so the comment would have to be updated.

Also it says that we don't do `maybe_fetches_limit()` and we still don't do them. Should we though?

@albert-llimos could you advise whether we can remove that comment / how to update it?